### ISSUE TITLE
Function Values in Keyframes

### ISSUE THREAD

**Issue by RobertAron** [cssinjs/jss#1216]
I've been stuck on this for a few days and I believe that it is a bug.

Thanks for making this awesome library!

__Expected behavior:__
Function values used in `@keyframes` do not appear to attach correctly to the style sheet.

__Describe the bug:__
The function is run, but not applied to the style sheet. Looking in chrome's inspector I can see that the keyframe is setup, but does not properties inside of it.

__Codesandbox link:__ 
https://codesandbox.io/s/react-jss-playground-5w8nc?fontsize=14

Basically this:
```js
"@keyframes my-animation": {
    "25%": {
      backgroundColor: ({ customColor }) => customColor
    }
}
```

__Versions (please complete the following information):__
10.0.0

Based on issue https://github.com/cssinjs/jss/issues/471 I thought this problem was fixed. I could be using the tool incorrectly.

**Comment by jasonsoden** [comment]
Same issue here. We really need to support values coming in via props.

**Comment by CodySchaaf** [comment]
Anyone able to come up with a workaround in the meantime?

**Comment by ashkanahrabi** [comment]
Any update on this? Still can't pass props to keyframes!

**Comment by pavelbochkov** [comment]
Hello! 
I fixed this issue but my fix could be not very good.

in react-jss sheets.js:

do not create new "dynamic" keyframes rule which won't be used anywhere, just update already existing keyframes rules if they have dynamic values. 

```
for (const key in meta.dynamicStyles) {
    const initialRuleCount = sheet.rules.index.length
    if (key.includes('keyframes')) {
      const idKey = key.replace('@keyframes', 'keyframes').split(' ').join('-')
      const originalKeyframeRule = sheet.rules.index.find(rule => rule.key === idKey)
      sheet.updateOne(originalKeyframeRule, data)
    } else {
      const originalRule = sheet.addRule(key, meta.dynamicStyles[key])
      // Loop through all created rules, fixes updating dynamic rules
      for (let i = initialRuleCount; i < sheet.rules.index.length; i++) {
        const rule = sheet.rules.index[i]
        sheet.updateOne(rule, data)
        // If it's the original rule, we need to add it by the correct key so the hook and hoc
        // can correctly concat the dynamic class with the static one
        rules[originalRule === rule ? key : rule.key] = rule
      }
    }
  }
```

I hope someone more experienced can improve on this.
